posts/1-vitrina/index.html

```diff
@@ -0,0 +1,165 @@
+<head>
+    <meta charset="UTF-8"/>
+    <link rel="stylesheet" href="reset.css"/>
+    <link rel="stylesheet" href="layout.css"/>
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="icon" href="/assets/favicon.ico"/>
+    <title>ביקורת נחפמים - ויטרינה</title>
+</head>
+
+<body>
+    <video class="background" autoplay loop muted>
+        <source src="/assets/1/background.mp4" type="video/mp4"/>
+    </video>
+
+    <main class="wrapper">
+        <div class="middle">
+            <h1>ביקורת ויטרינה</h1>
+            <div class="preamble">
```

Comment:
Is there a way to make the preamble bigger on phones?

Reply:
Yep, apparently it was oddly sized because of something called "font boosting" (which I've disabled now)